lxd/instance/drivers/driver_lxc.go

```diff
@@ -2786,6 +2786,25 @@ func (c *lxc) Shutdown(timeout time.Duration) error {
 	return nil
 }
 
+// Restart restart the instance.
+func (c *lxc) Restart(timeout time.Duration) error {
+	if timeout == 0 {
+		if err := c.Stop(false); err != nil {
```

Comment:
LXD coding style is to stay away from the one line defintion+check syntax.
So this should be kept as it was before on two lines, same for the ones below.